_data/team.yml

```diff
@@ -115,6 +115,15 @@ en: &DEFAULT_EN
         social:
           - url: https://github.com/rs-sprout98
             icon: fab fa-github
+      name: Aysha Luis
+      role: "Fellow in Clinical AI"
+      team-member-page: /team_member/aysha.html
+      image: assets/img/team/aysha.jpg
+      social:
+        - url: https://twitter.com/ayshaluis
+          icon: fab fa-twitter
+        -
```

Comment:
Similarly, this stray hyphen is breaking the code when I try to build it.

I can fix this, but it would be a good exercise for you to do it!